### PR TITLE
Prac 7 feedback

### PR DESCRIPTION
**Pull request by ChristineYamin**
@TinNilarSoe 
@ivy-naw-mth
please check my code

### CODE REVIEW

File: prac_07/project_management.py [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+                project['completed'] = int(completion_percentage) >= 100
+            if priority:
+                project['priority'] = priority
+            print("Project updated successfully!")
+            return
+    print("Project not found.")
+
+main()
+
+
```
Comment (TinNilarSoe) [inline prac_07/project_management.py:100]: Prompts are not consistent with sample output so I think you need to recheck according to instruction.

File: prac_07/my_guitars.py [hunk 2]
```diff
@@ -27,4 +26,23 @@ def read_data_from_file(FILENAME):
 def results_guitars(guitars):
     for guitar in guitars:
         print(guitar)
+def get_userinput(guitars):
+    """ Getting input from the user"""
+    name = input("Name : ")
```
Comment (TinNilarSoe) [inline prac_07/my_guitars.py:31]: Better to use .title() for name and also to include error-checking for inputs.

File: README.md [hunk 3]
```diff
@@ -1,3 +1,26 @@
...
+Writing a clean code is a fundamental skill for every software developer.
+Here are 3 main lessons that i have learned  about clean code in this subjects.
+-Don't repeat yourself (DRY)
+-Avoid Magic Numbers and Strings
+-Docstrings and Comments
+
+## Links to the Programming Patterns page and the CP1404 Practicals instructions repo.
+https://github.com/CP1404/Starter/wiki/Programming-Patterns
+https://github.com/CP1404/Practicals/tree/master
+
```
Comment (ivy-naw-mth) [inline README.md:25]: Such a good read me, good work! ^^

File: prac_07/language_file_reader.py [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+def using_csv_namedtuple():
+    """Language file reader version using both csv module and named tuple."""
+    Language = namedtuple('Language', 'name, typing, reflection, year')
+    in_file = open("languages.csv", "r")
+    in_file.readline()
+    for language in map(Language._make, csv.reader(in_file)):
+        print(language.name, 'was released in', language.year)
+        print(repr(language))
+
+#using_csv_namedtuple()
```
Comment (TinNilarSoe) [inline prac_07/language_file_reader.py:90]: Good code!

File: prac_07/project_management.py (hunk 1 above)
Comment (ChristineYamin) [reply prac_07/project_management.py:100]: i will fix that , Thanks you

File: prac_07/my_guitars.py (hunk 2 above)
Comment (ChristineYamin) [reply prac_07/my_guitars.py:31]: I will add error checking

File: README.md [hunk 5]
```diff
@@ -1,3 +1,26 @@
-# CP1404 Practicals
+# CP1404 practicals
+
+## Practical work for IT@JCU CP1404
+
+## Brief information about the repo and yourself.
+
+
+My name is Shwe Yamin Oo ,and I am studying data science major in James Cook University, Singapore.Among the subjects that i took in this trimester, CP1404 is a subject that I am most interested.
```
Comment (Anastasia-Eremenko) [inline README.md:8]: Nice and clear formatting, includes relevant info

Comment (ChristineYamin) [reply README.md:8]: Thanks you

**Pull request merged**